getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,182 @@
+
+.. prompt:: bash
+
+   butler query-collections /repo/dc2 --chains=tree "2.2i/*"
+   butler query-collections /repo/dc2 --chains=tree "u/mrawls/*"
+
+These commands should print a list of collections that meet the search criteria.
+The "tree chain" view is a handy way to see the "chained" data products that are included with each collection, such as calibs, skymaps, refcats, etc.
+
+To see some available datasets for processing, try, e.g.,
+
+.. prompt:: bash
+
+   butler query-data-ids /repo/dc2 tract patch visit --collections='2.2i/defaults/test-med-1' --where "skymap='DC2' AND band='g' AND tract=3828 AND patch=47 AND instrument='LSSTCam-imSim'"
+
+This command should print a list of data IDs that meet the search criteria, along with their tract, patch, and visit number.
+
+Processing Data
+===============
+
+Now it's time to process some data.
+In this guide, we will start with raws, run "processCcd" (which includes :py:mod:`lsst.ip.isr.IsrTask`, :py:mod:`lsst.pipe.tasks.characterizeImage.CharacterizeImageTask`, and :py:mod:`lsst.pipe.tasks.calibrate.CalibrateTask`), and make good seeing coadd templates.
+
+In a second pipeline, we will run difference imaging using the templates we just built and save the results in an Alert Production Database (APDB).
+
+Building good seeing templates
+------------------------------
+
+Here is the pipeline we will use:
+
+.. prompt:: yaml
+
+   description: An AP pipeline for building templates with LsstCam-imSim data
+
+   instrument: lsst.obs.lsst.LsstCamImSim
+   imports:
+   - location: $AP_PIPE_DIR/pipelines/ApTemplate.yaml
+   tasks:
+     calibrate:
+       class: lsst.pipe.tasks.calibrate.CalibrateTask
+       config:
+         connections.astromRefCat: 'cal_ref_cat_2_2'
+         connections.photoRefCat: 'cal_ref_cat_2_2'
+         astromRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         photoRefObjLoader.ref_dataset_name: 'cal_ref_cat_2_2'
+         python: |
+         config.astromRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+         config.photoRefObjLoader.filterMap = {band: 'lsst_%s_smeared' % (band) for band in 'ugrizy'};
+   subsets:
+     singleFrameAp:
+     subset:
+       - isr
+       - characterizeImage
+       - calibrate
+       - consolidateVisitSummary
+     description: >
+       Tasks to run for single frame processing that are necessary to use the good seeing selector to build coadds for use as difference imaging templates.
+
+This example pipeline imports a pipeline from :py:mod:`lsst.ap.pipe` you may `view on GitHub <https://github.com/lsst/ap_pipe/blob/master/pipelines/ApTemplate.yaml>`__.
+There are some special configurations concerning reference catalogs that must be set for this camera and/or dataset, so the example pipeline above lists the ``calibrate`` task explicitly to add custom configurations.
+
+To run this example pipeline, save it as ``ApTemplate-DC2.yaml``, choose an appropriate output collection name, and run
```

Comment:
Suggested change:
...choose an appropriate output collection name (u/USERNAME/OUTPUT-COLLECTION-1 in the example below)...